src/main/java/com/crowd/service/ProjectService.java

```diff
@@ -125,7 +125,9 @@ public void checkProjectStatus(Project project) {
 
     public ProjectResponse getProjectById(int projectId){
         ProjectResponse projectResponse = new ProjectResponse(projectDAO.findById(projectId));
-        projectResponse.isEditable(authService.isSameWithAuthUser(userDAO.findById(projectResponse.getUserId())));
+        boolean isSameUser = authService.isSameWithAuthUser(userDAO.findById(projectResponse.getUserId()));
+        boolean isProjectInProgress = projectResponse.isProjectInProgress();
+        projectResponse.isEditable(isSameUser && !isProjectInProgress);
```

Comment:
- [ ] It's strange to get data from project response and set data in it according to that data. Please suggest more incapsulated approach. 
- [ ] rename method to `setEditable` according to java bean convention